packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -290,45 +327,71 @@ export class OfficeAddinUsageData {
    * "Successful fail" means that there was an error as a result of user error, but our code worked properly
    * @param projectName Project name sent to Application Insights
    * @param data Data object(s) sent to Application Insights
+   * @deprecated Use `reportExpectedError` instead.  
    */
   public sendUsageDataSuccessfulFailEvent(method: string, data: object = {}) {
     this.sendUsageDataEvent({
       Succeeded: true,
       Method: method,
       Pass: false,
       ...data
-    });
+    });  
   }
 
   /**
-   * Reports custom event object to Application Insights
+   * Reports custom exception event object to Application Insights
    * @param projectName Project name sent to Application Insights
+   * @param err Error or message about error sent to Application Insights
    * @param data Data object(s) sent to Application Insights
+   * @deprecated Use `reportUnexpectedError` instead.  
    */
-  public sendUsageDataEvent(data: object = {}) {
```

Comment:
I think this function should remain as the basic function for sending the telemetry event. The other reportSuccess() or reportError0- function can compose the data object and then call this function. We only need one place in which the actual sending is done.

Reply:
It still exists!
It is just that the changes are getting pretty long and github is showing the changes in quite a weird way.

I will tag you in where you can find the function. I didn't even touch it.